slick/src/main/scala/slick/jdbc/MySQLProfile.scala

```diff
@@ -94,6 +94,12 @@ trait MySQLProfile extends JdbcProfile { profile =>
         else l
       }
     }
+
+    //Reference: https://github.com/slick/slick/issues/1419
+    override def createTableNamer(meta: MTable): TableNamer = new TableNamer(meta){
+      override def schema = meta.name.catalog
+      override def catalog = meta.name.schema 
```

Comment:
It would be helpful if you can go into more details on the cause of this issue. You can probably provide your analysis under the corresponding issue. This would help other maintainers and future contributors to better understand what happened.